assets/js/popup.js

```diff
@@ -0,0 +1,136 @@
+  const customElement = document.createElement(element);
+  if (children !== undefined) {
+    children.forEach((el) => {
+      if (el.nodeType) {
+        if (el.nodeType === 1 || el.nodeType === 11) customElement.appendChild(el);
+      } else {
+        customElement.innerHTML += el;
+      }
+    });
+  }
+  addAttributes(customElement, attributes);
+  return customElement;
+};
+// Print modal-bg and modal content containers
+const printModal = (content) => {
+  const modalContent = createCustomElement('div', {
+    id: 'modal-content',
+    class: 'modal',
+  }, [content]);
+
+  const modalContainer = createCustomElement('div', {
+    id: 'modal-container',
+    class: 'modal-bg',
+  }, [modalContent]);
+
+  document.body.appendChild(modalContainer);
+};
+
+const modalProject = [
+  {
+    title: 'Tonic',
+    img: 'assets/images/desktop-images/desktop-work1.png',
+    description: 'A daily selection of privately perosonalized reads; no accounts or sign-ups required.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'assets/images/desktop-images/desktop-work2.png',
+    description: 'Experimental content creation feature that allows users toadd to an existing story over the course of a day without spamming their friends.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Facebook 360',
+    img: 'assets/images/desktop-images/desktop-portfolio-image1.png',
+    description: 'Exploring the future of media in Facebooks first VirtualReality app; a place to discover and enjoy 360 photos andvideos on Gear VR.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Uber Navigation',
+    img: 'assets/images/desktop-images/desktop-portfolio-image2.png',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  }];
+// Modal projects event
+// First Modal
+projectBtn.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    printModal(`<div class="modal-header-container">
+    <div class="modal-title-container">
+      <h2 class="modal-title">${modalProject[index].title}</h2></div>
+      <div class="modal-header-icon">
+        <i id ="close-menu" class="fas fa-times"></i>
+      </div>
+    </div>
+    <div class="modal-subtitle-container">
+      <ul class="modal-mark">
+        <li>
+          <p>CANOPY</p>
+        </li>
+        <li>
+          <p>Back End Dev</p>
+        </li>
+        <li>
+          <p>2015</p>
+        </li>
```

Comment:
- [ ] Please add the dividing circles between the tags: 

Your design  | Figma design
------------- | -------------
<img width="248" alt="Screenshot 2022-04-07 at 13 09 32" src="https://user-images.githubusercontent.com/77038610/162164072-17330d42-d95f-4e83-ab21-1437e98316cc.png"> | <img width="225" alt="Screenshot 2022-04-07 at 13 09 47" src="https://user-images.githubusercontent.com/77038610/162164142-0b37ea51-2a2d-4a60-8f31-8bfcee6502f2.png">